pkgs/tools/networking/wuzz/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "wuzz";
```

Comment:
The version must be embedded in the name, as in `wuzz-${version}`. The `version` attribute is not special; version strings are actually extracted from `name` using `parseDrvName`.